api/build.gradle

```diff
@@ -179,6 +179,7 @@ buildscript {    // Configuration for building
     GAE_VERSION = '1.9.64'
     SPRING_BOOT_VERSION = '1.5.22.RELEASE'
     JACKSON_VERSION = '2.9.10'
```

Comment:
nit: any reason not to just bump it across the board?

Reply:
The `.1` bump is a Databind-only security release.